ci/infra/azure/terraform.tfvars.example

```diff
@@ -0,0 +1,117 @@
+# AZURE location
+# A list of location names can be found here:
+# https://azure.microsoft.com/en-in/global-infrastructure/locations/
+# https://azure.microsoft.com/en-us/global-infrastructure/regions/
+# https://docs.microsoft.com/en-us/azure/availability-zones/az-region
+# azure_location = "West Europe"
+azure_location = "East Asia"
+
+# Enable zone for available regions
+enable_zone = false
+
+# CIRD blocks for virtual network
+cidr_block = "10.1.0.0/16"
+
+# CIDR blocks for the bastion host subnet
+bastionhost_subnet_cidr = "10.1.1.0/24"
+
+# Private subnet of virtual_network
+private_subnet_cidr = "10.1.4.0/24"
+
+# Identifier to make all your resources unique and avoid clashes with other users of this terraform project
+# stack_name = "my-sles-cluster"
+
+# Name of DNS domain
+# dnsdomain = "my.domain.com"
+dnsdomain = "my-cluster-101.example.com"
+
+# Enables creation of bastion host
+# https://docs.microsoft.com/en-us/azure/bastion/bastion-overview
+# create_bastionhost = true
+create_bastionhost = false
```

Comment:
To comment or not to comment, that's the question? :smile:

Drop one of them, I would personally pick the **not** commented line.

Reply:
changed